modules/ROOT/pages/package-a-mule-application.adoc

```diff
@@ -0,0 +1,193 @@
+= Mule Maven Plugin Packaging
+
+You can package your Mule applications using Mule Maven Plugin(command line) or Anypoint Studio. See xref:studio::import-export-packages.adoc#exporting-projects-from-studio[Exporting Projects from Studio] for details about this procedure.
+
+Mule applications are packaged into a deployable JAR file that you can later deploy to a running Mule runtime engine. The package contains the application and all its dependencies whether those be JAR files required by the application or by its plugins.
+
+Every Mule application has two descriptors. These descriptors define your Mule application's dependencies:
+
+* A `mule-artifact.json` file. +
+This file describes how your application is composed. +
+
+* A `pom.xml` file. +
+This file describes all the dependencies required by the package to work properly.
+
+== Compatibility
+
+Mule 4 applications are compatible with the following Anypoint Studio and Mule Maven Plugin versions:
+
+* Anypoint Studio 7.x
+* Mule Maven Plugin 3.x
+
+== Add the Mule Maven Plugin to a Mule Project
+
+include::partial$mmp-concept.adoc[tag=addMuleMavenPluginToAProject]
+
+== Package a Mule Application
+
+. Make sure you added the Mule Maven plugin to your `pom.xml` file.
+. From the command line in your project's folder, execute the package goal:
++
+[source,console,linenums]
+----
+mvn clean package
+----
+
+The plugin packages your application and creates the deployable JAR file into the target directory within your project's folder. +
+
+[[package-goal]]
+=== Package Goal
+
+The `package` goal generates the application JAR file. This goal binds by default to the http://maven.apache.org/ref/3.6.1/maven-core/lifecycles.html[Maven lifecycle phase]: `package`.
+
+Optional Parameters
+[%header%autowidth.spread,cols="a,a,a"]
+|===
+|Name | Type | Description
+|`onlyMuleSources` | boolean | Generates the application JAR file containing only the source code. This property is just for sharing purposes.
+|`attachMuleSources` | boolean | Attaches the source code inside the generated JAR file.
+|`lightweightPackage` | boolean | Doesn't generate the repository with all the application dependencies inside the JAR file. This property is just for sharing purposes.
+|===
+
+For example, to execute the `package` goal and set the `attachMuleSources` parameter, you run the following command:
+[source,console,linenums]
+----
+mvn package -DattachMuleSources
+----
+
+== Create an Application Package to Import into Anypoint Studio
+
+To generate a JAR file that can be imported into Anypoint Studio, you need to package your application using the `-DattachMuleSources` parameter to include source files and metadata that Studio requires to re-import the deployable file as an open Mule project into your workspace.
```

Comment:
I would break up this long sentence into two.
To generate a JAR file that can be imported into Anypoint Studio, you need to package your application using the `-DattachMuleSources` parameter.
Using this parameters includes source files and metadata that Studio requires to re-import the deployable file as an open Mule project into your workspace.